src/index.ts

```diff
@@ -56,22 +54,49 @@ export default function (
               .replace(whitespace, '-');
           }
 
+          const getActiveDocumentId = function() {
+            const findDocId = /docs-doc-id-(.*)?$/;
+            var match = $("html").attr("class").match(findDocId);
+            return match[1];
+          };
+
           const getPdfPath = function() {
             var pdfPath = document.location.pathname;
             if (pdfPath.endsWith('/')) {
               pdfPath = pdfPath.substr(0, pdfPath.length-1);
             }
-            const checkRootDoc = new RegExp("/docs(((\/[0-9]+\.[0-9]+)|\/next)?)$");
-            if (!checkRootDoc.test(pdfPath)) {
-              // ends with document (not with /docs or /docs/x.x or /docs/next), remove document 
-              var lastSlashPos = pdfPath.lastIndexOf('/');
-              pdfPath = pdfPath.substr(0, lastSlashPos);
+            var id = getActiveDocumentId();
+
+            while (!pdfPath.endsWith(id) && id != "" && id.includes("/")) {
```

Comment:
All of this is sketchy at best, this should probably be a theme plugin so we have access to the actual ids.